sea-orm-migration/src/migrator.rs

```diff
@@ -307,6 +307,109 @@ pub trait MigratorTrait: Send {
 
         Ok(())
     }
+
+    /// Apply or Rollback migrations to version
+    async fn change_to_version(db: &DbConn, version: &str) -> Result<(), DbErr> {
+        Self::up_to_version(db, version).await?;
+        Self::down_to_version(db, version).await
+    }
```

Comment:
I think this shouldn't perform `up` then `down`. Instead, it should go in one direction only